ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -129,3 +138,60 @@ def send_metrics_from_properties(self, properties, metrics_map, prefix, tags):
                     )
                     return
                 self.send_metric(metric_type, metric_full_name, metric_value, new_tags)
+
+    def run_with_isolation(self):
+        # Lazy import required for monkey patching
+        from datadog_checks.base.checks.base import aggregator, datadog_agent
+
+        message_indicator = os.urandom(8).hex()
+        instance = dict(self.instance)
+
+        # Prevent fork bomb
+        instance['process_isolation'] = False
+
+        env_vars = dict(os.environ)
+        env_vars['REPLAY_MESSAGE_INDICATOR'] = message_indicator
+        env_vars['REPLAY_INSTANCE'] = json.dumps(instance)
+        env_vars['REPLAY_INIT_CONFIG'] = json.dumps(self.init_config)
+        env_vars['REPLAY_CHECK_ID'] = self.check_id
+
+        process = subprocess.Popen(
+            [
+                sys.executable,
+                '-u',
+                '-c',
+                'from datadog_checks.ibm_mq.replay import main;main()',
+            ],
+            stdin=subprocess.PIPE,
```

Comment:
is there any stdin to pipe?

Reply:
it's for the process to retrieve values from `datadog_agent`